### ISSUE TITLE
CC005 - wrong behavior

### ISSUE THREAD

**Issue by AlbertoMonteiro** [apigee/apigeelint#566]
Error

```
║ 166      │ 21       │ error    │ Invalid Condition - cannot parse                       │ CC005
```

Code

```xml
<Condition>(recaptcha-valid != true) and (recaptcha-score &lt;= 0.7)</Condition>
```

<img width="1877" height="52" alt="Image" src="https://github.com/user-attachments/assets/6061f9f9-5b24-44af-ab77-a74842a6ef9d" />

I also tried `LesserThanOrEquals`, but didnt fixed.

```
apigeelint -V
2.67.0
```

**Comment by ssvaidyanathan** [comment]
@DinoChiesa - TruthTable failing when we use a decimal
